raddb/sites-available/coa-relay

```diff
@@ -123,24 +123,20 @@ server coa {
 		#  what these attributes are unfortunately, so we
 		#  cannot give more detailed advice here.
 		#
-		update control {
-
-			#
-			#  Example MySQL lookup
-			#
-#			&Tmp-String-0 := "%{sql:SELECT IFNULL(GROUP_CONCAT(CONCAT(nasipaddress,'#',acctsessionid) separator '|'),'') FROM (SELECT * FROM radacct WHERE ('%{User-Name}'='' OR UserName='%{User-Name}') AND ('%{Acct-Session-Id}'='' OR acctsessionid = '%{Acct-Session-Id}') AND AcctStopTime IS NULL) a}"
-
-			#
-			#  Example PostgreSQL lookup
-			#
-#			&Tmp-String-0 := "%{sql:SELECT STRING_AGG(CONCAT(nasipaddress,'#',acctsessionid),'|') FROM (SELECT * FROM radacct WHERE ('%{User-Name}'='' OR UserName='%{User-Name}') AND ('%{Acct-Session-Id}'='' OR acctsessionid = '%{Acct-Session-Id}') AND AcctStopTime IS NULL) a}"
+		#
+		#  Example MySQL lookup
+		#
+#		&control.Tmp-String-0 := "%{sql:SELECT IFNULL(GROUP_CONCAT(CONCAT(nasipaddress,'#',acctsessionid) separator '|'),'') FROM (SELECT * FROM radacct WHERE ('%{User-Name}'='' OR UserName='%{User-Name}') AND ('%{Acct-Session-Id}'='' OR acctsessionid = '%{Acct-Session-Id}') AND AcctStopTime IS NULL) a}"
 
-			#
-			#  Keep a count of what we send.
-			#
-			&Tmp-Integer-0 := 0
+		#
+		#  Example PostgreSQL lookup
+		#
+#		&control.Tmp-String-0 := "%{sql:SELECT STRING_AGG(CONCAT(nasipaddress,'#',acctsessionid),'|') FROM (SELECT * FROM radacct WHERE ('%{User-Name}'='' OR UserName='%{User-Name}') AND ('%{Acct-Session-Id}'='' OR acctsessionid = '%{Acct-Session-Id}') AND AcctStopTime IS NULL) a}"
```

Comment:
Hmm... with a bit more work in `src/lib/unlang/edit.c`, I could probably just allow for:

`&control.Tmp-String-0 := %{sql:SELECT ...}`  (no `"..."`)

and then if there are multiple values returned, it creates multiple `&control.Tmp-String-0`.

But for now this is OK.